packages/pass-style/src/types.d.ts

```diff
@@ -162,6 +162,12 @@ export type RemotableObject<I extends InterfaceSpec = string> = PassStyled<
   I
 >;
 
+/**
+ * Abstract remotable method names to its own type in preparation of restricting
+ * it in a later PR.
+ */
+export type RemotableMethodName = string | symbol;
```

Comment:
I believe we’re in agreement that this will be narrowed to strings except "@@asyncIterator" and specifically Symbol.asyncIterator, somehow, at least at runtime if not here.

Reply:
Yes. Resolving.

Reply:
While we're agreed, I'm leaving this further narrowing to a later PR since this one is still not trying to enforce anything.